pages/index.js

```diff
@@ -12,29 +12,27 @@ export default function Home() {
       <Background />
       <Navbar />
       <div className={styles.main}>
-        <div className={styles.mainText}>
-          <Text style="title">
-            Bem vindo(a) ao <Logo style="logoExtraLarge">Rate.io</Logo>
+        <div className={styles.main__text}>
+          <Text style="text__title">
```

Comment:
isn't that?
```suggestion
          <Text className={styles.text__title}>
```

Reply:
we are sending the syle as props so the class names can be handled inside the text component file.
I guess if we leave as you suggested it will trigger the className from the index scss file